cert_manager/datadog_checks/cert_manager/cert_manager.py

```diff
@@ -0,0 +1,32 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.checks.openmetrics import OpenMetricsBaseCheck
+
+from .metrics import METRIC_MAP
+
+
+class CertManagerCheck(OpenMetricsBaseCheck):
+    DEFAULT_METRIC_LIMIT = 0
+    HEALTH_METRIC = 'cert_manager.prometheus.health'
+
+    def __init__(self, name, init_config, agentConfig, instances=None):
```

Comment:
```suggestion
    def __init__(self, name, init_config, instances=None):
```
`agentConfig` is part of the Agent 5 signature, which we've deprecated.